dockstore-webservice/src/main/java/io/dockstore/webservice/core/Group.java

```diff
@@ -53,7 +54,7 @@
 @Entity
 @Table(name = "usergroup")
 @NamedQueries(@NamedQuery(name = "io.dockstore.webservice.core.Group.findAll", query = "SELECT t FROM Group t"))
-public class Group  implements Comparable<Group> {
+public class Group  implements Comparable<Group>, Serializable {
```

Comment:
Might just want to delete this since we didn't end up going down this route with sharing

Reply:
Do you mean delete the entire entity?

Reply:
Yes, I would do a safe-delete and check what references this class. 
But I think since we went the SAM integration route, this class is actually used by anything.